src/plugins/data_view_management/public/components/field_editor/components/field_format_editor/field_format_editor.tsx

```diff
@@ -23,7 +23,7 @@ export interface FieldFormatEditorProps {
   fieldType: string;
   fieldFormat: FieldFormat;
   fieldFormatId: string;
-  fieldFormatParams: { type?: string | undefined } & FieldFormatParams;
+  fieldFormatParams: FieldFormatParams<{ type?: string }>;
```

Comment:
It seems better for `FieldFormatParams` to be generic, instead of the previous way